.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  extends: 'kartotherian',
+  env: {
```

Comment:
this needs to be bumped to the latest eslint kartotherian config - won't need either "env" or "rules" section.